include/freetds/tds.h

```diff
@@ -1324,6 +1324,21 @@ TDSICONV *tds_iconv_from_collate(TDSCONNECTION * conn, TDS_UCHAR collate[5]);
 
 
 /* mem.c */
+typedef struct tds_tvp_row
+{
+	TDSPARAMINFO * params;
+	struct tds_table_row * next;
+} TDS_TVP_ROW;
+
+typedef struct tds_tvp
+{
+	char * schema;
+	char * typename;
+	int num_cols;
+	TDS_TVP_ROW * metadata;
+	TDS_TVP_ROW * row;
+} TDS_TVP;
+
```

Comment:
`TDS_TVP`/`TDS_TVP_ROW` structs - stores table and row information in TDS format